src/main/resources/db/migration/common/V2024.11.13.0001__correct_indexes_on_review_and_review_schedule_tables.sql

```diff
@@ -0,0 +1,24 @@
+--- review table indexes
+DROP INDEX review_prison_number_idx;
```

Comment:
The 2 indexes were previously created incorrectly in `V2024.11.09.0009__add_review_and_review_schedule_tables`:
```
CREATE UNIQUE INDEX review_prison_number_idx ON review
(
    prison_number
);

...

CREATE UNIQUE INDEX review_schedule_prison_number_idx ON review
(
    prison_number
);
```
In that there was a unique index on `review.prison_number` which would have meant a prisoner could only have 1 review
And the unique index that was supposed to be on `review_schedule.prison_number` was actually also on `review` !!
So neither indexes did what they were supposed to! 🤦‍♂️